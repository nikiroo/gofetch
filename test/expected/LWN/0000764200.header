0[$] Writing network flow dissectors in BPF	null/LWN/0000764200		70
i  [Kernel] Sep 6, 2018 15:59 UTC (Thu) (corbet)
i
i    Network  packet  headers  contain  a great deal of information,
i    but  the  kernel  often only needs a subset of that information
i    to  be  able to perform filtering or associate any given packet
i    with  a  flow.  The  piece  of  code that follows the different
i    layers  of  packet  encapsulation to find the important data is
i    called  a  flow  dissector.  In current Linux kernels, the flow
i    dissector  is  written  in  C.  A  patch  set has been proposed
i    recently  to  implement  it  in  BPF  with  the  clear  goal of
i    improving security, flexibility, and maybe even performance.
i
