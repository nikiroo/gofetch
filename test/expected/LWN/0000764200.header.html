<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h2><a href='0000764200.html'>[$] Writing network flow dissectors in BPF</a></h2>
	<div class='details'>([Kernel] Sep 6, 2018 15:59 UTC (Thu) (corbet))</div>
	<br/>
	<div class='content' style='text-align: justify'>
		Network packet headers contain a great deal of information, but the kernel often only needs a subset of that information to be able to perform filtering or associate any given packet with a flow. The piece of code that follows the different layers of packet encapsulation to find the important data is called a flow dissector. In current Linux kernels, the flow dissector is written in C. A patch set has been proposed recently to implement it in BPF with the clear goal of improving security, flexibility, and maybe even performance.
	</div>
<hr/>
</div>
</body>
